lib/module.js

```diff
@@ -110,3 +110,5 @@ module.exports = async function gtmModule (_options) {
 }
 
 module.exports.meta = require('../package.json')
+
+module.exports.defaultSettings = defaults
```

Comment:
Maybe would better to create a `defaults.js` file so we can directly test it but not exposing to outside.

Reply:
Yes. You are right. It makes sense.